src/api/ingest/__init__.py

```diff
@@ -1,12 +1,10 @@
-from .serializers import *
-from .benthiclit_serializer import *
-from .benthicpit_serializer import *
-from .benthicpqt_serializer import *
-from .bleaching_serializer import *
-from .fishbelt_serializer import *
-from .habitatcomplexity_serializer import *
-
-from .attributes_ingester import BenthicIngester, FishIngester
+from .attributes_ingester import BenthicIngester, FishIngester  # noqa: F401
```

Comment:
F401  is needed for those imports because the classes aren't being used in this python module, unlike the reset are being used in `ingest_serializers`